dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -283,8 +283,8 @@ Configuration InstallCAM
 				$retrycount = 1800
 				while ($retryCount -gt 0)
 				{
-					$readyToConfigure = ( Get-Item "Registry::HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{26A24AE4-039D-4CA4-87B4-2F86418091F0}"  -ErrorAction SilentlyContinue )
-					# don't wait for {64A3A4F4-B792-11D6-A78A-00B0D0180910} - that's the JDK. The JRE is installed 2nd {26A...} so wait for that.
+					$readyToConfigure = ( Get-Item "Registry::HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{26A24AE4-039D-4CA4-87B4-2F64180144F0}"  -ErrorAction SilentlyContinue )
+					# don't wait for {64A3A4F4-B792-11D6-A78A-00B0D0180144} - that's the JDK. The JRE is installed 2nd {26A...} so wait for that.
```

Comment:
So 26A24AE4-039D-4CA4-87B4-2F64180144F0 is the new JRE? Surprised the first few sections didn't change but if so that's okay.

Reply:
Yes, it is for the new JRE, I think the last part 2F64180144F0 changes for each build. Yesterday you saw the error Java Installation was too long error. I went to VM and check the registration and found the ID was changed. I deployed again use this new value and it successed

Reply:
Okay, thanks.